installers/windows.gradle

```diff
@@ -48,11 +48,10 @@ task agentWindowsExe(type: WindowsPackagingTask, dependsOn: [':agent-bootstrappe
     project.copy { from('go-agent/release/start-agent.bat') into(versionedDir()) }
     project.copy { from('go-agent/release/stop-agent.bat') into(versionedDir()) }
     project.copy { from('go-agent/release/README.md') into(versionedDir()) }
-    project.copy { from('go-agent/win/cruisewrapper.exe') into(versionedDir()) }
+    project.copy { from("windows-shared/tanuki/${flavour()}/cruisewrapper.exe") into(versionedDir()) }
```

Comment:
Are we building a 64bit and 32 bit installers? Or just separate 32 bit installers with 32 bit and 64 bit bundled jre?